### DISCUSSION TITLE
XLD 配置与抓轨指南

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#15] [Announcements]
# XLD 配置与抓轨指南

在 macOS 平台，XLD 是一款与 EAC 同样优秀的抓轨软件，本文将详细介绍其使用方法。将日期设置为 2018-05-27 是为了纪念本文的初成。

https://zexwoo.blog/posts/tutorials/xld-ripping/

**Comment by hifiboy2018** [comment]
亲爱的博主你好。很喜欢您分享的内容，最近也在使用XLD和EAC，但我的光驱容错率似乎不理想，经常出现读盘卡死，希望您能推荐1-2款适合抓轨的驱动，万分感谢。

**Reply by ZexWoo** [reply]
光驱没啥驱动可言。如果你想要好的光驱型号，可以参考我的一个小伙伴做的统计表：https://yasra.notion.site/yasra/f68f87af9b78476ea3695ed27808dc84

**Reply by hifiboy2018** [reply]
好的，谢谢。

**Reply by hifiboy2018** [reply]
进去看了，型号非常多，越靠前的型号越推荐吗？

**Reply by ZexWoo** [reply]
是的，一般推荐排名靠前的，体验会比较好。

**Comment by hifiboy2018** [comment]
这是一封自动回复邮件。已经收到您的来信，我会尽快回复。

**Comment by Mmx88** [comment]
大佬，参照此教程进行到抓取CD的那一页设置偏离值那边是0，点小三角提示未找到已注册光驱，也勾选了自动设置，抓轨完看log也是偏离值0，是这个光驱不支持吗，用的是macbookpro接转接头插上古早的那个苹果官方外置光驱，

**Reply by Mmx88** [reply]
补充一下，已找到办法从eac上测出光驱为hldtst gx50n，到网站上查表得到是lge的gx50n，偏移值为6，与windows的eac下抓轨后log上的6一致，但是在mac的xld上就是没法检测出来需要手动填

**Reply by ZexWoo** [reply]
XLD 可以手动填写，应该也能启用 AccurateRip，所以问题不大。

**Comment by hifiboy2018** [comment]
这是一封自动回复邮件。已经收到您的来信，我会尽快回复。

**Comment by 3ange** [comment]
请教一下 我是sony ad7260s 支持c2。有必要勾选c2吗

**Reply by ZexWoo** [reply]
否。有些光驱会谎报自己支持，但事实上不支持。

**Reply by 3ange** [reply]
再请教一个问题 我有符合内嵌尺寸的专辑插图 在配置里如何修改让音频文件内嵌封面呢  勾选「将专辑插图嵌入文件」然后把符合内嵌尺寸的cover.jpg放在文件夹内  就可以了嘛

**Reply by ZexWoo** [reply]
我没用过 XLD 的嵌入封面功能，你可以抓轨完成后用其它工具完成这项任务。

或者你直接测试性地操作一下，如果好使就更好了。

**Reply by Mmx88** [reply]
可以试试kid3，我就用这个管理信息标签和添加封面歌词

**Reply by 3ange** [reply]
直接import musicbrain就可以吧 然后自己校对一下 没问题保存。不会对我抓轨的音频文件造成负面影响吧

**Reply by ZexWoo** [reply]
修改元数据不会改变音频数据。

**Comment by hifiboy2018** [comment]
这是一封自动回复邮件。已经收到您的来信，我会尽快回复。

**Comment by 3ange** [comment]
再请教一下 抓整轨wav的话 是不是只要通用里改成wav。然后抓轨的时候选择整个文件+cue就可以了

**Reply by ZexWoo** [reply]
没错儿。

**Reply by 3ange** [reply]
谢谢解答
